assets/js/mbta.js

```diff
@@ -887,6 +929,21 @@ const arincToRealtimeIdMap = {
   'GARL-m': 'arlington_mezzanine',
   'GBOY-e': 'boylston_eastbound',
   'GBOY-w': 'boylston_westbound',
+  'GPRKE-e': 'park_st_east_platform_eastbound',
```

Comment:
These four Park Street zones need to be identified because none of them is really what they seem to be... in fact maybe I'll go down to Park Street this afternoon and you can send test messages